hello.lua

```diff
@@ -10,11 +12,12 @@ local config = {
 local client = ld.clientInit(config, 1000)
 
 local user = ld.makeUser({
-    key = "abc"
+    key = "example-user-key",
+    name = "Sandy"
 })
 
```

Comment:
```suggestion
local value = client:boolVariation(user, YOUR_FEATURE_KEY, false)
print("feature flag "..YOUR_FEATURE_KEY.." is "..tostring(value).." for this user")
```
Not 100% sure this will work, but we can check the CI results.